### ISSUE TITLE
[Bug]: User password reset does not work with user id

### ISSUE THREAD

**Issue by mattamon** [pimcore/pimcore#15883]
### Pimcore version

11.0 - 11.x

### Steps to reproduce

Try to reset user password using command `pimcore:user:reset-password` and use user id as input e.g.
`bin/console pimcore:user:reset-password 2`

Using the user name works e.g. `pimcore:user:reset-password admin`

### Actual Behavior

Following error message is shown: 

```bash
bin/console pimcore:user:reset-password 2

In AbstractUser.php line 41:
                                                                                                                                                                                                           
  Pimcore\Model\User\AbstractUser::getById(): Argument #1 ($id) must be of type int, string given, called in /var/www/html/vendor/pimcore/pimcore/bundles/CoreBundle/src/Command/ResetPasswordCommand.php  
   on line 63                                                                                                                                                                                              
                                                                                                                                                                                                           

pimcore:user:reset-password [-p|--password [PASSWORD]] [--] <user>
```

### Expected Behavior

Password input should be requested and no error message is shown

**Comment by blankse** [comment]
@mattamon I created a PR for this https://github.com/pimcore/pimcore/pull/15884

**Comment by dvesh3** [comment]
Fixed by #15884
